functions/src/api/v1/reminders/editReminder.ts

```diff
@@ -0,0 +1,109 @@
+import { Request, Response } from 'express'
+import { wrap } from 'async-middleware'
+import { assertRightToEditRoom } from '../../../db/assertRightsToEditRoom'
+import {
+    ReminderEditData,
+    updateReminderDb,
+    getReminderDb,
+} from '../../../db/remindersDb'
+import { isDestinationsCorrectlyFormatted } from '../utils/isDestinationsCorrectlyFormatted'
+import { BadRequestError } from '../../errors/HttpError'
+import { Timestamp } from '../../../firebase/firebase'
+import { assertTimestampInFuture } from './assertTimestampInFuture'
+
+/**
+ * @swagger
+ * /v1/rooms/{roomId}/reminders/{reminderId}:
+ *   patch:
+ *     description: Update an existing reminder, either to change the destinations (all at once, replacing them) or the datetime at which it will be sent.
+ *     tags:
+ *       - rooms
+ *     consumes:
+ *     - multipart/form-data
+ *     produces:
+ *     - application/json
+ *     parameters:
+ *       - name: hostName
+ *         description: The name or organisation which sent the invite(s)
+ *         in: x-www-form-urlencoded
```

Comment:
what do you think about API accepted content types (for requests) @ntchou ? We discussed the subject with @HugoGresse , and are wondering if to accept both
- `x-www-form-urlencoded`
- `json`

I also wonder if it would be better to only accept `json` format, so that the request and response types are consistent. Furthermore, using the first type seems to always require having to parse (`JSON.parse`) some stuff from within the routes, whereas with `json` type the express middleware should take care of this

Reply:
though json it used a lot, we may not forget that maybe some company/institut using IV will not be as modern, may use old java system for eg, and form was the way to go